src/main/java/at/hannibal2/skyhanni/config/features/misc/MiscConfig.java

```diff
@@ -283,4 +283,9 @@ public class MiscConfig {
     @ConfigEditorBoolean
     @FeatureToggle
     public boolean maintainGameVolume = false;
+
+    @Expose
+    @ConfigOption(name = "Magical Power Display", desc = "")
+    @Accordion
+    public MPDisplayConfig mpDisplay = new MPDisplayConfig();
```

Comment:
move into inventory category, rename the variable and class name to MagicalPower